packages/skatejs/src/with-update.js

```diff
@@ -241,6 +245,9 @@ export const withUpdate = (Base: Class<any> = HTMLElement): Class<any> =>
       if (super.connectedCallback) {
         super.connectedCallback();
       }
+      keys(this.constructor.props).forEach(prop => {
```

Comment:
Why do all props need to be signalled as changing when connected?

Reply:
This would be effectively the same as firing a default `attributeChangedCallback` on connecting, so my thought here was that we want to react to all the values. Usually we need the values for something, so we need to use them all initially. In my cases for example, I need to set WebGL rendering properties/features based on the initial values.

There isn't a way to know if `updated` is being called for the firs time, and the `updated` method will generally always contain conditional checks to determine which props changed, so in order to fire logic for all props, this is why I thought to initially set all props modified.

If we don't set them all as "modified" initially, then the only option we have is to avoid the use of `modifiedProps` in the `updated` method and instead diff `oldProps` vs `this.props`, which defeats the purpose of having `modifiedProps`.

Is there maybe a better way to run initial logic for all props?